### ISSUE TITLE
Rounded corners are a bit rough

### ISSUE THREAD

**Issue by AaronTechnic** [yilozt/mutter-rounded#40]
![](https://cdn.discordapp.com/attachments/845964267520917545/937609712038473748/Screenshot_from_2022-01-31_05-15-30.png)
 As you can see from the screenshot, the rounded corners are a bit rough.

**Comment by Diegovsky** [comment]
Yeah, that's not a problem with the code. The code simply "cuts" the window corner, so drawing a little border around it like the terminal does would be quite the hack.

**Comment by SilverMarcs** [comment]
Does using the border-width property help? you can access it via `mutter_settings` or dconf-editor

**Comment by Microwave-WYB** [comment]
Try to make clipping on each edge 1px in `mutter_settings`, this will effectively remove the original borders provided by your GTK Theme. If you still want the border, set the border width in `mutter_settings`

**Comment by diazabdulm** [comment]
Honestly, mutter should ship with those borders clipped (I personally have them at 1,1,2,1). Solves what some may think is a bug, and it looks nicer without those horrendous default 1px solid black borders on otherwise light-themed apps in Gnome. But I think the visuals speak for themselves:
![imgonline-com-ua-twotoone-fxikUd19mlkOW](https://user-images.githubusercontent.com/20374641/176811691-8ecf956f-a953-4593-b2aa-f2cbe7cdee65.jpg)

**Comment by aspizu** [comment]
Is it possible to add a feature to render additional borders on windows?
